Automattic/themes
# Varia: fix button styles leaking into editor

This is a hotfix for https://github.com/Automattic/wp-calypso/issues/50332.

As a follow up I will also test https://github.com/WordPress/gutenberg/pull/29229 to see if it resolves the issue.

D57440-code

**To test**
- Check out this PR onto your sandbox
- Activate Hever or another Varia theme
- Verify the button styles don't leak into the editor UI
